app/controllers/ApplicationController.scala

```diff
@@ -137,8 +138,8 @@ class ApplicationController @Inject()(val applicationService: ApplicationService
             subscribedVersions = subscribedContexts.map(filterOutVersions)
             subscribedWithFields = subscribedVersions.map(filterForFields)
 
-            seqOfSubscriptions = subscribedVersions.values.toSeq.flatMap(asSeqOfSeq)
-            subscriptionsThatHaveFieldDefns = subscribedWithFields.values.toSeq.flatMap(asSeqOfSeq)
+            seqOfSubscriptions = subscribedVersions.values.toSeq.flatMap(asSeqOfSeq).sortWith(_._1 < _._1)
```

Comment:
Are you sure you don't want to create an implicit Ordering for this and just use the .sorted method.

Reply:
We didn't think it would be reused.